packages/@n8n/benchmark/package.json

```diff
@@ -5,6 +5,8 @@
   "main": "dist/index",
   "scripts": {
     "build": "tsc -p tsconfig.build.json && tsc-alias -p tsconfig.build.json",
+    "lint": "eslint . --quiet",
```

Comment:
There is currently no flood of warnings as this is a new package, so we can remove `--quiet` to prevent hiding any warnings as they come up.